### PR TITLE
Testgrid add unique test id for test instance

### PR DESCRIPTION
**Pull request by emosbaugh**
#### What type of PR is this?

type::chore

#### What this PR does / why we need it:

kurlUrl is not unique in ui so ui is all mixed up. see screenshots from refreshing the same page twice.

Screen Shot 2022-06-22 at 10 44 58 AM
Screen Shot 2022-06-22 at 10 44 49 AM

#### Which issue(s) this PR fixes:

Fixes NONE

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Does this PR require documentation?

NONE

### CODE REVIEW

File: testgrid/web/package.json [hunk 1]
```diff
@@ -47,9 +48,8 @@
     "webpack": "^5.64.4",
     "webpack-bundle-analyzer": "^4.4.2",
     "webpack-cli": "^4.9.1",
-    "webpack-dev-server": "^4.9.0",
-    "webpack-merge": "^5.8.0",
-    "axios": "^0.27"
+    "webpack-dev-server": "^3.11.3",
```
Comment (emosbaugh) [inline testgrid/web/package.json:51]: i had to revert this to get it to work

File: testgrid/tgapi/pkg/persistence/pg.go [hunk 2]
```diff
@@ -20,12 +20,6 @@ func MustGetPGSession() *sql.DB {
 		panic(err)
 	}
 
-	err = db.Ping()
```
Comment (emosbaugh) [inline testgrid/tgapi/pkg/persistence/pg.go:23]: skaffold was not starting

Comment (laverya) [reply testgrid/tgapi/pkg/persistence/pg.go:23]: are you sure you should remove the db ping though? Doesn't that just run a `select 1` etc?

Comment (emosbaugh) [reply testgrid/tgapi/pkg/persistence/pg.go:23]: we dont use this pattern in any other of our projects

**Review by laverya — DISMISSED** [review]

**Review by laverya — APPROVED** [review]

**Pull request merged**